theme/webpack.prod.js

```diff
@@ -185,7 +185,7 @@ module.exports = merge(
 						files: ['request_page.hbs'],
 						rules: [
 							{
-								replace: 'https://customer.liferay.com/project-details',
+								replace: 'https://customer.liferay.com/group/customer/project-details',
```

Comment:
Looks good, we probably want to remove `/group/customer/` in favor of the friendly url here.

Reply:
I will submit a change to rectify that. Also going to update the link on Customer Help Center to use the new url changed here.